src/Types/EnumType.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+
+final class EnumType extends Type
```

Comment:
Why do you make the class final? Other types are not final

Reply:
Well e.g. my custom enum types could extend it, and then it'd be clear hierarchy, that this is an enum related type. Its not critical, but there is just one other type (`AsciiStringType`) that is final, so I don't see reason why limit the inheritance specifically for `EnumType`, all other types are open.

Reply:
You don't need that hierarchy. This class contains a single method which is just a delegate to a platform method. Extending this class is absolutely pointless.

And we can discuss finalizing the other types for the sake of consistency as well, if you like. 😉